### DISCUSSION TITLE
Do you use this library or command  line program?

### DISCUSSION THREAD

**Discussion by dougransom** [dougransom/vix_utils#14] [General]
If you are are comfortable, please share how you are using vix_utils in your project.

**Comment by AQuant206** [comment]
Initially using as command line utility to learn & pull historical data.  Think I've figured out the new command line options except -r analogy to use with src\examples\sample_plots.py .

Once I've understood it and confirm it meets my needs, I'd like to use python library to add daily close futures values for cash term structure.
